docs/install.rst

```diff
@@ -66,6 +66,11 @@ We recommend that you use `Homebrew <https://brew.sh/>`_::
 
     brew install python3
 
+**Docker**
```

Comment:
This is in the wrong section (prerequisites)? I think this should go in the section `Installing Rally` just below `PyEnv Install`?

Reply:
Right thanks for spotting. Fixed in https://github.com/elastic/rally/pull/702/commits/af1a12402329af7bf27343bc33d63d3fef7abd4b